glib/src/param_spec.rs

```diff
@@ -898,6 +1072,32 @@ impl ParamSpecOverride {
             from_glib_none((*ptr).overridden)
         }
     }
+
+    pub fn builder<'a>(name: &'a str, overridden: &'a ParamSpec) -> ParamSpecOverrideBuilder<'a> {
```

Comment:
As I mentioned before, I don't see the point of this builder as both params are required

Reply:
Doesn't hurt IMHO. Maybe it's useful for someone and it's at least consistent.